### PR TITLE
IRC bot improvements

### PR DESCRIPTION
**Pull request by tkdrg**
Adds a sleep before nickserv because apparently there is such a thing as
going too fast. prints messages to stderr and uses stdout for the raw
socket input instead to ease debugging.

### CODE REVIEW

**Comment by Iamgoofball** [comment]
i argue that this should be fastmerged since it's not gameplay related at all, and this is the bot's maintainer. No need for a 24 hour wait period, not like this is gonna effect gameplay at all.

**Comment by tkdrg** [comment]
mso basically did this locally already, so it doesn't really matter to be honest.

**Comment by optimumtact** [comment]
should've waited for the MOTD

I tried to warn you bro, but you didn't listennn

**Comment by tkdrg** [comment]
Too much complexity too little gain

**Comment by optimumtact** [comment]
I tried this approach, the sleep just kept growing and growing in length until I gave in and just listened for those event codes.

File: bot/minibot.py [hunk 1]
```diff
@@ -50,12 +56,13 @@ def nudge_handler(irc):
...
                 s, ip = nudge.accept()
             except:
-                print("Nudge socket lost, attempting to reopen.")
+                print_err("Nudge socket lost, attempting to reopen.")
                 nudge = setup_nudge_socket()
                 continue
             rawdata = s.recv(1024)
             s.close()
             data = pickle.loads(rawdata)
+            print(data)
```
Comment (optimumtact) [inline bot/minibot.py]: was this left in on purpose?

File: bot/minibot.py [hunk 2]
```diff
@@ -73,6 +80,7 @@ def irc_handler(irc):
         try:
             buf = irc.recv(1024).decode("UTF-8").split("\n")
             for i in buf:
+                print(i)
```
Comment (optimumtact) [inline bot/minibot.py]: left in on purpose as well?

Comment (tkdrg) [inline bot/minibot.py]: Yes for both. MSO asked for this in order to debug it more easily when he remotes in.

**Comment by optimumtact** [comment]
someone merge this ffs

**Pull request merged**